core/src/main/java/org/apache/accumulo/core/iteratorsImpl/conf/ColumnSet.java

```diff
@@ -30,6 +30,10 @@
 import org.apache.accumulo.core.util.Pair;
 import org.apache.hadoop.io.Text;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+@SuppressFBWarnings(value = "CT_CONSTRUCTOR_THROW",
+    justification = "Constructor validation is required for proper initialization")
```

Comment:
While we may be reluctant to make final public API classes that the user might have extended, stuff in `*Impl` are fair game. I'm not sure if it's possible to make them all `final` (we might extend them ourselves, elsewhere internally), but if it is possible, there shouldn't be any harm doing so.